markdown_it/common/utils.py

```diff
@@ -141,9 +141,10 @@ def replaceEntityPattern(match, name):
 
 
 def unescapeMd(string: str):
-    if "\\" in string:
-        return string
-    return string.replace(UNESCAPE_MD_RE, "$1")
+    raise NotImplementedError
+    # if "\\" in string:
+    #     return string
+    # return string.replace(UNESCAPE_MD_RE, "$1")
```

Comment:
I simply commented this function out, because `str.replace()` takes strings as input, not re.Patterns, so this is probably broken.